public/sass/_variables.light.scss

```diff
@@ -60,7 +60,7 @@ $page-bg:  		  		    $white;
 $body-color:    		    $gray-1;
 $text-color:    		    $gray-1;
 $text-color-strong:	    $white;
-$text-color-weak: 	    $gray-1;
+$text-color-weak: 	    $gray-2;
```

Comment:
@torkelo yes it was. Either we solve it at this level or we have to create a new variable for https://github.com/grafana/grafana/blob/master/public/sass/components/_gf-form.scss#L21 not sure which is better.